### DISCUSSION TITLE
Nothing is visible (dark font on dark background)

### DISCUSSION THREAD

**Discussion by saporator** [brackets-cont/brackets#312] [General]
HI all,

since quite some time I have with the laest version (Release 2.2 Build 2.2.0-17942  on macOS) the issue that I can't see anything. The backgroudn is dark and also the font is in the same color. Only if I open a menu comand e.g. About, or View - Design I'm able to see (until the popup is shown) the files, contact in the background. As soon as I close the popup it's dark again.
I toggled between Dark and Light design but this doesn't help.

Any idea what I can do?

**Comment by charlypa** [comment]
could you share screen shot

**Comment by saporator** [comment]
![Bildschirmfoto 2024-01-29 um 19 18 48](https://github.com/brackets-cont/brackets/assets/59700253/16df9feb-0b8d-4963-b8a5-347643a980fd)

**Comment by charlypa** [comment]
@saporator 
Apologies for the late reply. Hope this issue is fixed with Phoenix Code.
We've updated our editor to Phoenix Code, launched just yesterday. It now supports Windows, Mac, and Linux moving beyond the older Brackets x86 binaries.
 You can download Phoenix Code at https://phcode.io/.

**Comment by saporator** [comment]
@charlypa 
Is seems that Phoenix code is a different app? The link on the mentioned web page returns a html file which looks not very trustworthy...
When I reinstall Brackets it's working for sometime, then again nothing is visible.

**Reply by abose** [reply]
Phoenix Code is made by the Brackets Team itself and is a different app. It is an IDE built on top of brackets and the platform that powers the next generation of Brackets editor.

**Comment by charlypa** [comment]
@saporator 
`Phoenix Code` is the latest addition to the brackets community. Phoenix Code is built by the same team that has released brackets.

**Comment by saporator** [comment]
@abose @charlypa 
Thank you.It was my mistake to have the wrong platform selected.
I'm testing PHoenix Code now.
